### PR TITLE
Close header line creator once ingest is complete [VS-1045]

### PR DESCRIPTION
**Pull request by mcovarr**
Simple copy/paste bug. Closing the header line creator fixes the hanging issues as seen in [this run](https://job-manager.dsde-prod.broadinstitute.org/jobs/21c1ec08-444e-4acd-8490-cc9640d9ea03) (requires PMI ops).

Integration run [in progress](https://job-manager.dsde-prod.broadinstitute.org/jobs/3b5129bb-b7fe-47db-abc4-dda5d7f5006a) (regular auth).

### CODE REVIEW

File: src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/CreateVariantIngestFiles.java [hunk 1]
```diff
@@ -383,7 +383,7 @@ public void closeTool() {
             vetCreator.closeTool();
         }
         if (vcfHeaderLineScratchCreator != null) {
-            vetCreator.closeTool();
+            vcfHeaderLineScratchCreator.closeTool();
```
Comment (gbggrant) [inline src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/CreateVariantIngestFiles.java:386]: Nice find!

File: scripts/variantstore/wdl/GvsUtils.wdl [hunk 2]
```diff
@@ -72,7 +72,7 @@ task GetToolVersions {
     # there are a handlful of tasks that require the larger GNU libc-based `slim`.
     String cloud_sdk_slim_docker = "gcr.io/google.com/cloudsdktool/cloud-sdk:435.0.0-slim"
     String variants_docker = "us.gcr.io/broad-dsde-methods/variantstore:2023-09-27-alpine"
-    String gatk_docker = "us.gcr.io/broad-dsde-methods/broad-gatk-snapshots:varstore_2023_09_13"
+    String gatk_docker = "us.gcr.io/broad-dsde-methods/broad-gatk-snapshots:varstore_2023_09_27_4a061605"
```
Comment (gbggrant) [inline scripts/variantstore/wdl/GvsUtils.wdl:75]: Note that I have a PR https://github.com/broadinstitute/gatk/pull/8531 in which I have built and tagged `us.gcr.io/broad-dsde-methods/broad-gatk-snapshots:varstore_2023_09_27` with my updates, so if you need to use that tag, please rebuild / repush / retag.

**Review by gbggrant — APPROVED** [review]
Integration test plz.

**Review by gbggrant — COMMENTED** [review]
Oops. Meant to request the integration test before approval

**Comment by mcovarr** [comment]
>Integration test plz.

Hmm, the existing integration test does not exercise `process_vcf_headers` so it wouldn't test these changes. However, in the spirit of "the VDS integration tests should exercise what we intend to use for AoU", perhaps at least the VDS integration tests should be modified to exercise this.

**Comment by gbggrant** [comment]
Yeah, that's a good point.

**Review by gbggrant — APPROVED** [review]
Looks good to me. Thanks for updating the integration test.
Okay to merge once the tests pass.

**Comment by mcovarr** [comment]
Latest greatest integration success https://job-manager.dsde-prod.broadinstitute.org/jobs/4ccc1f23-fcf4-4549-8046-a52364ec80c3

File: src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/CreateVariantIngestFiles.java (hunk 1 above)
Comment (rsasch) [reply src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/CreateVariantIngestFiles.java:386]: d'oh!

**Review by rsasch — APPROVED** [review]
thanks for finding this (not sure why it didn't cause problems before...?)

**Pull request merged**